spread.yaml

```diff
@@ -497,11 +497,11 @@ suites:
      - ubuntu-18.04*
    environment:
      SNAPCRAFT_PACKAGE_TYPE: "snap"
+     CRYPTOGRAPHY_DONT_BUILD_RUST: "1"
    prepare: |
      apt-get update
      apt-get install --yes gcc g++ make python3-dev python3-pip python3-wheel libffi-dev libsodium-dev libapt-pkg-dev squashfs-tools xdelta3 bzr git mercurial subversion libxml2-dev libxslt-dev
-     pip3 install --user --upgrade wheel setuptools
-     pip3 install --user --upgrade pip==20.1
+     pip3 install --user --upgrade cryptography==3.4 wheel setuptools pip==20.1
```

Comment:
```suggestion
     pip3 install --user --upgrade wheel setuptools
     pip3 install --user --upgrade pip==20.1
     pip3 install --user --upgrade cryptography==3.4
```

Reply:
But this will break, setuptools depends on cryptography...